client_test.go

```diff
@@ -21,7 +21,7 @@ func TestClient(t *testing.T) {
 	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		switch {
 		case r.Method == http.MethodGet && r.URL.Path == "/ping":
-			r.Write(bytes.NewBufferString("pong"))
+			w.Write([]byte("pong"))
```

Comment:
Is this a mistake?
maybe, you intended to write "ping" to response instead of request

Reply:
oh... Thanks for fixing 😅